client/src/components/Overview/Overview.jsx

```diff
@@ -3,12 +3,18 @@ import PropTypes from 'prop-types';
 import axios from 'axios';
 import ImageGallery from 'Overview/ImageGallery.jsx';
 import ProductInformation from 'Overview/ProductInformation.jsx';
+import StyleSelector from 'Overview/StyleSelector.jsx';
+import AddToCart from 'Overview/AddToCart.jsx';
 import testData from 'tests/testData.js';
 
 function Overview({ productId }) {
-  const [product, setProduct] = useState({});
-  const [styles, setStyles] = useState([]);
-  const [selectedStyleId, setSelectedStyleId] = useState();
+  // const [product, setProduct] = useState({});
+  // const [styles, setStyles] = useState([]);
+  // const [selectedStyleId, setSelectedStyleId] = useState();
+  //TODO: delete once data fetching is up
+  const [product, setProduct] = useState(testData.product);
+  const [styles, setStyles] = useState(testData.styles.results);
```

Comment:
Just a note, if you make the request in this file you will need to account for the time where that data isn't added yet so it wont error out

Reply:
Won't we have to do this anyways? Even if we fetch data at the top level, the initial state will still need to be accounted for.